tests/Fixer/Operator/OperatorLinebreakFixerTest.php

```diff
@@ -190,6 +190,60 @@ function foo()
                 prepare_value:
                 $objectsPool[$value] = [$id = \count($objectsPool)];
         '];
+
+        yield 'handle ignored operator' => [
```

Comment:
This test isn't really testing the new feature - without `['ignored' => ['->']]` the changes will be the same.

Let's add 2 chains - one with `->` at the beginning and another with `->` at the end to show both are not touched (as before this PR one would be changed).